### PR TITLE
Bugfix on Stat priority no content

### PR DESCRIPTION
**Pull request by OHaiDanny**
This PR fixes the issue where we hide the link to the stat priority page if there is no content.

image

### CODE REVIEW

File: layouts/partials/job/gearing.html [hunk 1]
```diff
@@ -21,15 +21,25 @@
...
+            {{ partial "cards/default.html" $content }}
+          {{ else }}
+            {{ $content := dict
+              "role" $role
+              "icon" "/theme-assets/materia.svg"
+              "name" "Stats & Materia"
+              "text" .Params.priority
+            }}
+            {{ partial "cards/default.html" $content }}
+          {{ end }}
```
Comment (OHaiDanny) [inline layouts/partials/job/gearing.html:42]: I kinda dont like how im repeating code. I was trying to find out if Hugo has a way to set something like a "Global Variable" so I just have to update that in the IF statement...

File: layouts/partials/cards/default.html [hunk 2]
```diff
@@ -15,8 +15,10 @@
     <div>
       {{ .text }}
     </div>
-    {{ if isset . "readMoreLink" }}
-      {{ partial "cards/_footer.html" . }}
-    {{ end }}
+    <div class="border-t border-card-dark pt-5 text-right">
+      {{ if isset . "readMoreLink" }}
```
Comment (OHaiDanny) [inline layouts/partials/cards/default.html:19]: Moving the container outside this file will create an empty space when we don't render a link in the `dict`

File: layouts/partials/job/gearing.html (hunk 1 above)
Comment (nonowazu) [reply layouts/partials/job/gearing.html:42]: I think you should do a parent dict and then call "merge" on the extra options perhaps? Docs [here](https://gohugo.io/functions/merge/)

**Review by nonowazu — CHANGES_REQUESTED** [review]

File: layouts/partials/job/gearing.html (hunk 1 above)
Comment (OHaiDanny) [reply layouts/partials/job/gearing.html:40]: I found another solution to use `.Scratch` instead to solve it. the `merge` has a problem of scoping when it is inside if statements unfortunately...

https://www.regisphilibert.com/blog/2017/04/hugo-scratch-explained-variable/

**Review by nonowazu — APPROVED** [review]

**Pull request merged**